incipyt/tools/setuptools.py

```diff
@@ -95,9 +104,9 @@ def add_to_structure(self):
                 sanitizer=sanitizers.package,
             ),
             "python_requires": templates.StringTemplate(
-                ">={PYTHON_VERSION}",
+                ">={AUDIENCE_PYTHON_VERSION}",
                 sanitizer=sanitizers.version,
-                PYTHON_VERSION="3.7",
+                AUDIENCE_PYTHON_VERSION=audience_version(),
```

Comment:
As we only need that function here, we can also do:
```suggestion
                AUDIENCE_PYTHON_VERSION="{0[0]}.{0[1]}".format(min(sys.version_info, (3, 9))),  # noqa: FS002
```
I don't think we need that anywhere else.